homeassistant/components/verisure/config_flow.py

```diff
@@ -53,13 +54,34 @@ async def async_step_user(
             self.email = user_input[CONF_EMAIL]
             self.password = user_input[CONF_PASSWORD]
             self.verisure = Verisure(
-                username=user_input[CONF_EMAIL], password=user_input[CONF_PASSWORD]
+                username=self.email,
+                password=self.password,
+                cookieFileName=self.hass.config.path(
+                    STORAGE_DIR, f"verisure_{user_input[CONF_EMAIL]}"
+                ),
             )
+
             try:
                 await self.hass.async_add_executor_job(self.verisure.login)
             except VerisureLoginError as ex:
-                LOGGER.debug("Could not log in to Verisure, %s", ex)
-                errors["base"] = "invalid_auth"
+                if "Multifactor authentication enabled" in str(ex):
+                    try:
+                        await self.hass.async_add_executor_job(self.verisure.login_mfa)
+                    except (
+                        VerisureLoginError,
+                        VerisureError,
+                        VerisureResponseError,
+                    ) as mfa_ex:
+                        LOGGER.debug(
+                            "Unexpected response from Verisure during MFA set up, %s",
+                            mfa_ex,
+                        )
+                        errors["base"] = "unknown"
```

Comment:
Should this indicate to the user that it was an unknown error while trying to do MFA?